server/internal/core/infrastructure/grpc/proto/v1/runner_auth.proto

```
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package proto.v1;
```

Buf (1.70.0)

[error] Files with package "proto.v1" must be within a directory "proto/v1" relative to root but were in directory "server/internal/core/infrastructure/grpc/proto/v1".

(PACKAGE_DIRECTORY_MATCH)